src/internal/kopia/cleanup_backups_test.go

```diff
@@ -529,6 +625,173 @@ func (suite *BackupCleanupUnitSuite) TestCleanupOrphanedData() {
 			buffer:    24 * time.Hour,
 			expectErr: assert.NoError,
 		},
+		// Tests dealing with assist base cleanup.
+		{
+			name: "AssistBase NotYoungest InBufferTime Noops",
```

Comment:
Given these tests are testing some complex code behavior, could you please add some comments for each test for more context?

Reply:
Also, can we add a test which has both assist & merge bases? Just to establish the cleanup behavior that we are always going to keep the most recent assist base, even if it's superseded by a merge base.

Reply:
>can we add a test which has both assist & merge bases

Good callout! I've added a new test for that and added comments to all the assist base tests